src/styles/theme.scss

```diff
@@ -0,0 +1,384 @@
+
+/* ------------------------------
+
+   Variables
+
+------------------------------ */
+
+// Contemplating Action variables
```

Comment:
These will need to be the brand colours

Reply:
We could change them all to grey-scale at this stage but I'm not sure that's necessary.